src/ConnectedMode/UI/OrganizationSelection/OrganizationSelectionViewModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.Core.WPF;
+
+namespace SonarLint.VisualStudio.ConnectedMode.UI.OrganizationSelection;
+
+public class OrganizationSelectionViewModel : ViewModelBase
+{
+    private OrganizationDisplay selectedOrganization;
+
+    public OrganizationDisplay SelectedOrganization
+    {
+        get => selectedOrganization;
+        set
+        {
+            selectedOrganization = value;
+            RaisePropertyChanged();
+            RaisePropertyChanged(nameof(IsValidSelectedOrganization));
+        }
+    }
+
+    public bool IsValidSelectedOrganization => SelectedOrganization is { Key: var key } && !string.IsNullOrWhiteSpace(key) && !key.Any(char.IsWhiteSpace);
```

Comment:
The key's validation is identical to the one from ManualOrganizationSelectionViewModel. I think we should extract it into a commonplace and then reuse it in both places.

Reply:
Simplified validation, lmk if it still makes sense to have it separated